*** tests/A-02-06-M-ok.out	Fri Oct 18 19:27:35 2024
--- tests/A-02-06-M-ok.outhyp	Thu Oct 24 23:43:05 2024
***************
*** 18,30 ****
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: HABITAT|h1|nome2|12|2
! ÁRVORE|T1|Pinheiro 4|60|20|PERENE|GERARFOLHAS
! ÁRVORE|T3|Figueira|5|10|CADUCA|GERARFOLHAS
  HABITAT|h2|outro nome|122|0
! HABITAT|h3|nome1|1222|2
! ÁRVORE|T2|Oliveira|120|10|PERENE|GERARFOLHAS
! ÁRVORE|T4|Plátano|30|20|CADUCA|GERARFOLHAS
  Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
--- 18,26 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: HABITAT|h1|nome2|12|0
  HABITAT|h2|outro nome|122|0
! HABITAT|h3|nome1|1222|0
  Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
