*** tests/A-04-02-M-ok.out	2024-10-24 13:33:59.529462700 +0100
--- tests/A-04-02-M-ok.outhyp	2024-10-24 17:21:35.201995100 +0100
***************
*** 18,24 ****
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Identificador único do habitat: Área do habitat: Alterar área de habitat: Operação inválida: O habitat 'AAA' não existe.
  Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
--- 18,24 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Habitat IDNew Habitat AreaAlterar área de habitat: Operação inválida: O habitat 'AAA' não existe.
  Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
