*** tests/A-04-02-M-ok.out	Fri Oct 18 19:27:35 2024
--- tests/A-04-02-M-ok.outhyp	Wed Oct 23 17:09:41 2024
***************
*** 18,24 ****
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Identificador único do habitat: Área do habitat: Alterar área de habitat: Operação inválida: O habitat 'AAA' não existe.
  Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
--- 18,24 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Habitat IDNew Habitat AreaAlterar área de habitat: Operação inválida: O habitat 'AAA' não existe.
  Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
