*** tests/A-04-01-M-ok.out	Fri Oct 18 19:27:35 2024
--- tests/A-04-01-M-ok.outhyp	Fri Oct 25 00:02:48 2024
***************
*** 18,24 ****
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Identificador único do habitat: Área do habitat: Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
--- 18,24 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Habitat IDNew Habitat AreaGestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
