*** tests/A-08-02-M-ok.out	Fri Oct 18 19:27:35 2024
--- tests/A-08-02-M-ok.outhyp	Thu Oct 24 23:24:11 2024
***************
*** 10,26 ****
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
- Escolha uma opção: 96
- Menu Principal
- 1 - Criar ficheiro
- 2 - Abrir ficheiro
- 3 - Guardar ficheiro
- 4 - Avançar estação do ano
- 5 - Ver satisfação global
- 6 - Gestão de animais
- 7 - Gestão de funcionários
- 8 - Gestão de habitats
- 9 - Gestão de vacinas
- 10 - Consultas
- 0 - Sair
  Escolha uma opção: 
\ No newline at end of file
--- 10,13 ----
