*** tests/A-09-03-M-ok.out	Fri Oct 18 19:27:35 2024
--- tests/A-09-03-M-ok.outhyp	Fri Oct 25 08:31:01 2024
***************
*** 22,28 ****
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: Nome do animal: Identificador único da espécie: Identificador único do habitat: Registar animal: Operação inválida: O animal 'a2' já existe.
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
--- 22,28 ----
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: Nome do animal: Identificador único da espécie: Identificador único do habitat: Registar animal: Operação inválida: O funcionário 'a2' já existe.
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
