*** tests/A-09-03-M-ok.out	2024-10-24 13:33:59.641647800 +0100
--- tests/A-09-03-M-ok.outhyp	2024-10-24 17:21:55.438930500 +0100
***************
*** 22,28 ****
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: Nome do animal: Identificador único da espécie: Identificador único do habitat: Registar animal: Operação inválida: O animal 'a2' já existe.
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
--- 22,28 ----
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: Nome do animal: Identificador único da espécie: Identificador único do habitat: Registar animal: Operação inválida: O funcionário 'a2' já existe.
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
