*** tests/A-16-03-M-ok.out	Fri Oct 18 19:27:35 2024
--- tests/A-16-03-M-ok.outhyp	Thu Oct 24 11:20:47 2024
***************
*** 17,32 ****
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: Identificador único do funcionário: Nome do funcionário: Tipo do funcionário? (VET ou TRT) Registar funcionário: Operação inválida: O funcionário 'T1' já existe.
! Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
  3 - Atribuir responsabilidade
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: Identificador único do funcionário: Nome do funcionário: Tipo do funcionário? (VET ou TRT) Registar funcionário: Operação inválida: O funcionário 'V1' já existe.
! Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
  3 - Atribuir responsabilidade
--- 17,30 ----
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: Identificador único do funcionário: Nome do funcionário: Tipo do funcionário? (VET ou TRT) Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
  3 - Atribuir responsabilidade
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: Identificador único do funcionário: Nome do funcionário: Tipo do funcionário? (VET ou TRT) Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
  3 - Atribuir responsabilidade
***************
*** 34,40 ****
--- 32,40 ----
  5 - Calcular satisfação de funcionário
  0 - Sair
  Escolha uma opção: TRT|T1|Trt one
+ TRT|T1|nome
  VET|V1|Vet one
+ VET|V1|new v1
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
