*** tests/A-18-05-M-ok.out	Fri Oct 18 19:27:35 2024
--- tests/A-18-05-M-ok.outhyp	Thu Oct 24 20:42:31 2024
***************
*** 18,24 ****
  5 - Calcular satisfação de funcionário
  0 - Sair
  Escolha uma opção: TRT|T1|Trt one
! VET|V1|Vet one|C1,C2
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
--- 18,24 ----
  5 - Calcular satisfação de funcionário
  0 - Sair
  Escolha uma opção: TRT|T1|Trt one
! VET|V1|Vet one
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
***************
*** 26,32 ****
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: Identificador único do funcionário: Identificador único da responsabilidade: Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
  3 - Atribuir responsabilidade
--- 26,33 ----
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: Employee IDResponsibilityRetirar responsabilidade: Operação inválida: Responsabilidade (habitat ou espécie) 'C2' não atribuída ao funcionário 'V1'.
! Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
  3 - Atribuir responsabilidade
***************
*** 34,40 ****
  5 - Calcular satisfação de funcionário
  0 - Sair
  Escolha uma opção: TRT|T1|Trt one
! VET|V1|Vet one|C1
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
--- 35,41 ----
  5 - Calcular satisfação de funcionário
  0 - Sair
  Escolha uma opção: TRT|T1|Trt one
! VET|V1|Vet one
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
