*** tests/A-19-01-M-ok.out	Fri Oct 18 19:27:35 2024
--- tests/A-19-01-M-ok.outhyp	Fri Oct 25 10:27:29 2024
***************
*** 17,23 ****
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: Identificador único do funcionário: 20
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
--- 17,23 ----
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: Employee IDSatisfaction of employee V3: 20
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
