*** tests/A-20-02-M-ok.out	2024-10-24 13:33:59.818545500 +0100
--- tests/A-20-02-M-ok.outhyp	2024-10-24 17:22:36.510234500 +0100
***************
*** 16,29 ****
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Identificador da vacina: Nome da vacina: Identificadores das espécies que podem receber a vacina: Registar vacina: Operação inválida: A espécie 'E4' não existe.
! Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
--- 16,29 ----
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Identificador da vacina: Nome da vacina: Identificadores das espécies que podem receber a vacina: Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: VACINA|V1|nome|0|E4
! Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
