*** tests/A-21-13-M-ok.out	Fri Oct 18 19:27:35 2024
--- tests/A-21-13-M-ok.outhyp	Thu Oct 24 16:56:15 2024
***************
*** 16,30 ****
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Identificador da vacina: Identficador do veterinário a ministrar a vacina: Identificador único do animal: A vacina 'Vac1' não é apropiada para o animal 'AA3'.
! Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Identificador da vacina: Identficador do veterinário a ministrar a vacina: Identificador único do animal: A vacina 'Vac2' não é apropiada para o animal 'AA1'.
! Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
--- 16,28 ----
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Identificador do animal: Identificador da vacina: Identficador do veterinário a ministrar a vacina: Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Identificador do animal: Identificador da vacina: Identficador do veterinário a ministrar a vacina: Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
***************
*** 48,56 ****
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: ANIMAL|AA1|Patudo|E1|CONFUSÃO|H1
  ANIMAL|AA2|Patudo 2|E1|VOID|H1
! ANIMAL|AA3|Malhado|E2|CONFUSÃO|H1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
--- 46,54 ----
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: ANIMAL|AA1|Patudo|E1|VOID|H1
  ANIMAL|AA2|Patudo 2|E1|VOID|H1
! ANIMAL|AA3|Malhado|E2|VOID|H1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
***************
*** 69,72 ****
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
! Escolha uma opção: 
--- 67,70 ----
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
! Escolha uma opção: 
\ No newline at end of file
