*** tests/A-21-13-M-ok.out	Fri Oct 18 19:27:35 2024
--- tests/A-21-13-M-ok.outhyp	Fri Oct 25 16:16:43 2024
***************
*** 48,56 ****
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: ANIMAL|AA1|Patudo|E1|CONFUSÃO|H1
  ANIMAL|AA2|Patudo 2|E1|VOID|H1
! ANIMAL|AA3|Malhado|E2|CONFUSÃO|H1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
--- 48,56 ----
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: ANIMAL|AA1|Patudo|E1|VOID|H1
  ANIMAL|AA2|Patudo 2|E1|VOID|H1
! ANIMAL|AA3|Malhado|E2|VOID|H1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
***************
*** 69,72 ****
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
! Escolha uma opção: 
--- 69,72 ----
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
! Escolha uma opção: 
\ No newline at end of file
