*** tests/A-21-10-M-ok.out	Fri Oct 18 19:27:35 2024
--- tests/A-21-10-M-ok.outhyp	Fri Oct 25 21:10:51 2024
***************
*** 57,63 ****
  0 - Sair
  Escolha uma opção: ANIMAL|AA1|Patudo|E1|VOID|H1
  ANIMAL|AA2|Patudo 2|E1|NORMAL|H1
! ANIMAL|AA3|Malhado|E2|NORMAL|H1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
--- 57,63 ----
  0 - Sair
  Escolha uma opção: ANIMAL|AA1|Patudo|E1|VOID|H1
  ANIMAL|AA2|Patudo 2|E1|NORMAL|H1
! ANIMAL|AA3|Malhado|E2|ACIDENTE|H1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
