*** tests/A-24-03-M-ok.out	Fri Oct 18 19:27:35 2024
--- tests/A-24-03-M-ok.outhyp	Fri Oct 25 23:08:40 2024
***************
*** 40,47 ****
  3 - Actos médicos de um veterinário
  4 - Vacinações com problemas
  0 - Sair
! Escolha uma opção: Identificador único do funcionário: REGISTO-VACINA|Vac3|Vet1|E1
! Consultas
  1 - Animais de um habitat
  2 - Actos médicos num animal
  3 - Actos médicos de um veterinário
--- 40,46 ----
  3 - Actos médicos de um veterinário
  4 - Vacinações com problemas
  0 - Sair
! Escolha uma opção: Identficador do veterinário a ministrar a vacina: Consultas
  1 - Animais de um habitat
  2 - Actos médicos num animal
  3 - Actos médicos de um veterinário
