*** tests/A-25-01-M-ok.out	Sat Oct 19 15:14:49 2024
--- tests/A-25-01-M-ok.outhyp	Fri Oct 25 15:56:40 2024
***************
*** 137,144 ****
  3 - Actos médicos de um veterinário
  4 - Vacinações com problemas
  0 - Sair
! Escolha uma opção: REGISTO-VACINA|Vac2|Vet2|E2
! Consultas
  1 - Animais de um habitat
  2 - Actos médicos num animal
  3 - Actos médicos de um veterinário
--- 137,143 ----
  3 - Actos médicos de um veterinário
  4 - Vacinações com problemas
  0 - Sair
! Escolha uma opção: Consultas
  1 - Animais de um habitat
  2 - Actos médicos num animal
  3 - Actos médicos de um veterinário
