*** testsBoth/A-01-03-M-ok.out	Thu Oct  3 22:33:09 2024
--- testsBoth/A-01-03-M-ok.outhyp	Fri Oct 11 12:03:45 2024
***************
*** 12,17 ****
--- 12,18 ----
  0 - Sair
  Escolha uma opção: 2
  Ficheiro a abrir: app01.dat
+ Abrir ficheiro: Operação inválida: Problema ao abrir ficheiro: Erro a processar ficheiro app01.dat
  Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
***************
*** 34,40 ****
  6 - Mostrar árvores de habitat
  0 - Sair
  Escolha uma opção: 1
- HABITAT|AR1|Aldeia dos Macacos|20|0
  Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
--- 35,40 ----
