*** testsBoth/A-01-09-M-ok.out	Thu Oct  3 22:33:10 2024
--- testsBoth/A-01-09-M-ok.outhyp	Fri Oct 11 10:57:45 2024
***************
*** 18,29 ****
  4 - Calcular satisfação de animal
  0 - Sair
  Escolha uma opção: 1
! ANIMAL|A1|Farrusco 1|E1|VOID|H1
! ANIMAL|A2|Farrusco 2|E1|VOID|H1
! ANIMAL|A3|Farrusco 3|E1|VOID|H2
! ANIMAL|B1|Farrusco 4|E1|VOID|H2
! ANIMAL|X1|Farrusco 5|E1|VOID|H3
! ANIMAL|X2|Farrusco 6|E1|VOID|H3
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
--- 18,29 ----
  4 - Calcular satisfação de animal
  0 - Sair
  Escolha uma opção: 1
! ANIMAL|A1|Farrusco 1|H1|VOID|E1
! ANIMAL|A2|Farrusco 2|H1|VOID|E1
! ANIMAL|A3|Farrusco 3|H2|VOID|E1
! ANIMAL|B1|Farrusco 4|H2|VOID|E1
! ANIMAL|X1|Farrusco 5|H3|VOID|E1
! ANIMAL|X2|Farrusco 6|H3|VOID|E1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
