*** testsBoth/A-01-17-M-ok.out	Thu Oct  3 22:33:10 2024
--- testsBoth/A-01-17-M-ok.outhyp	Fri Oct 11 11:58:15 2024
***************
*** 19,29 ****
  5 - Calcular satisfação de funcionário
  0 - Sair
  Escolha uma opção: 1
! VET|A24|Rohit Figueiredo
! VET|A34|Rohit Figueiredo 2
! VET|B24|Rohit Figueiredo 3
! VET|D24|Rohit Figueiredo 4
! VET|X24|Rohit Figueiredo 5
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
--- 19,29 ----
  5 - Calcular satisfação de funcionário
  0 - Sair
  Escolha uma opção: 1
! VETERIN´ARIO|A24|Rohit Figueiredo
! VETERIN´ARIO|A34|Rohit Figueiredo 2
! VETERIN´ARIO|B24|Rohit Figueiredo 3
! VETERIN´ARIO|D24|Rohit Figueiredo 4
! VETERIN´ARIO|X24|Rohit Figueiredo 5
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
