*** testsBoth/A-01-07-M-ok.out	Thu Oct  3 22:33:10 2024
--- testsBoth/A-01-07-M-ok.outhyp	Fri Oct 11 11:09:56 2024
***************
*** 18,24 ****
  4 - Calcular satisfação de animal
  0 - Sair
  Escolha uma opção: 1
! ANIMAL|A1|Farrusco|E1|VOID|H1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
--- 18,24 ----
  4 - Calcular satisfação de animal
  0 - Sair
  Escolha uma opção: 1
! ANIMAL|A1|Farrusco|H1|VOID|E1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
