*** testsBoth/A-01-08-M-ok.out	Thu Oct  3 22:33:10 2024
--- testsBoth/A-01-08-M-ok.outhyp	Fri Oct 11 11:31:52 2024
***************
*** 18,28 ****
  4 - Calcular satisfação de animal
  0 - Sair
  Escolha uma opção: 1
! ANIMAL|A1|Farrusco 1|E1|VOID|H1
! ANIMAL|A2|Farrusco 2|E1|VOID|H1
! ANIMAL|A3|Farrusco 3|E1|VOID|H1
! ANIMAL|B1|Farrusco 4|E1|VOID|H1
! ANIMAL|X1|Farrusco 5|E1|VOID|H1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
--- 18,28 ----
  4 - Calcular satisfação de animal
  0 - Sair
  Escolha uma opção: 1
! ANIMAL|A1|Farrusco 1|H1|VOID|E1
! ANIMAL|A2|Farrusco 2|H1|VOID|E1
! ANIMAL|A3|Farrusco 3|H1|VOID|E1
! ANIMAL|B1|Farrusco 4|H1|VOID|E1
! ANIMAL|X1|Farrusco 5|H1|VOID|E1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
